modules/Microsoft.RecoveryServices/vaults/replicationAlertSettings/readme.md

```diff
@@ -0,0 +1,51 @@
+# RecoveryServices Vaults ReplicationAlertSettings `[Microsoft.RecoveryServices/vaults/replicationAlertSettings]`
```

Comment:
Heads up to regenerate the readme after suggested changes if accepted

Reply:
Updated